### ISSUE TITLE
Empty output if css class contains double dash (--) and a comment at the beginning

### ISSUE THREAD

**Issue by ollm** [clean-css/clean-css#1224]
### Environment

* clean-css version - `master clean-css@5.3.0  (git+https://github.com/clean-css/clean-css.git#40ca65807639e0470dab22585847a7582687167f)`:
* node.js version - `v14.19.3`
* operating system `Ubuntu 20.04.3`

### Configuration options

```js
var CleanCSS = require('clean-css');
var output = new CleanCSS({
  level: 2 // But it also happens with level 1
}).minify(css)
```

### Input CSS

```css
.commentRatingColor--chartreuse
{
	/* Comment */
	color: #5ab900;
}
```

### Actual output CSS

```css
empty
```

### Expected output CSS

```css
.commentRatingColor--chartreuse{color:#5ab900}
```

## Works

```css
.commentRatingColor--chartreuse
{
	color: #5ab900;
	/* Comment */
}
```

### Actual output CSS

```css
.commentRatingColor--chartreuse{color:#5ab900}
```

**Comment by jakubpawlowicz** [comment]
Looks like parsing is off. I'll fix it.

**Comment by jakubpawlowicz** [comment]
The fix is out in 5.3.2.
